emscripten.py

```diff
@@ -1153,7 +1159,7 @@ def finalize_output(metadata, post, funcs_js, need_asyncify, provide_fround, asm
        access_quote('asmGlobalArg'), the_global,
        shared_array_buffer,
        access_quote('asmLibraryArg'), sending,
-       "'use asm';" if not metadata.get('hasInlineJS') and settings['ASM_JS'] == 1 else "'almost asm';", 
+       "'use asm';" if not metadata.get('hasInlineJS') and settings['ASM_JS'] == 1 and not settings['CYBERDWARF'] else "'almost asm';",
```

Comment:
I'll back this out. There's another change I'm working on that emits metadata intrinsics. There's code for it in CallHandlers.h but it's currently behind an option flag that's not plumbed through. In that case it inserts function calls when some source variable is assigned like:

`metadata_llvm_dbg_value_local($jsvar, type_id, offset, DWARF_OP, "source variable name")`

I believe that would not validate with asm.js. My thinking is that there would be additional configuration options for emcc to control intrinsics emitting.